README.md

```diff
@@ -204,6 +204,7 @@ Found [Clone] 7 duplicated lines with 10 tokens:
     * `--commit-uuid` [default: latest commit of current git branch] - Set the commit UUID that will receive the results on Codacy
     * `--skip-commit-uuid-validation` [default: false] - Force using a commit UUID even if it doesn't belong to the current git branch.
     * `--upload` [default: false] - Request to push results to Codacy
+    * `--skip-ssl-verification` [default: false] - Skips the SSL certificate verification when communicating with the Codacy API
```

Comment:
Another (but very minor) suggestion, just to make the help string consistent with the other ones. I only noticed this now:

```suggestion
    * `--skip-ssl-verification` [default: false] - Skip the SSL certificate verification when communicating with the Codacy API
```